src/routes/docs/tutorials/nuxt/step-7/+page.markdoc

```diff
@@ -0,0 +1,222 @@
+---
+layout: tutorial
+title: Ideas page
+description: Add ideas from Appwrite database in your app.
+step: 7
+---
+
+With the methods in the `useIdeas`composable we can get some ideas to the home page for the users to interact with.
+We will use it in a form component so the logged in users can add their ideas, and in a list component to render the ten most recent ideas.
+We start with building the form.
+
+# Idea form {% #idea-form %}
+
+On the home page, the logged in users should be able to add their ideas to the Appwrite database.
+The form need a text field for filling in the title, a textarea for the description and a submit button.
+
+From the `components` directory, add the file `ÌdeasForm.vue` and add the following code.
+
+```vue
+// components/IdeasForm.vue
+
+<template>
+  <div>
+    <article class="container padding-0">
+      <h4 class="heading-level-4">Submit Idea</h4>
+      <form @submit.prevent="handleAddIdea" class="u-margin-block-start-16">
+        <ul class="form-list">
+          <!-- Title input field -->
+          <li class="form-item">
+            <label class="label">Title</label>
+            <input
+              type="text"
+              placeholder="Title"
+              v-model="addIdeaData.title.value"
```

Comment:
Formdata could be used to avoid creating refs and binding them. Especially as nothing else is being done with these.

Reply:
Fixed